packages/core/readme.md

```diff
@@ -0,0 +1,20 @@
+### 📚 [Documentation](https://docs.walletconnect.com/2.0/introduction/web3modal/about)
```

Comment:
Why are we repeating the same readme across multiple sub projects, wouldn't it be more useful to have dedicated readmes that describe what this package does?

Reply:
Mainly for it to look nicer on npm